releasenotes/notes/images-doc-fix-7398899572eac510.yaml

```diff
@@ -0,0 +1,3 @@
+other:
+  - |
+    **[IMS]** Minor fix in``docs/data-sources/images_image_v2.md`` (`#1903 <https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/pull/1903>`_)
```

Comment:
From user perspective it's fix of `opentelekomcloud_images_image_v2` documentation